indico/modules/events/layout/util.py

```diff
@@ -349,6 +379,41 @@ def get_css_url(event, force_theme=None, for_preview=False):
         return _build_css_url(layout_settings.get(event, 'theme'))
 
 
+def _build_js_url(theme):
+    if ':' not in theme:
+        return None
+    try:
+        path = get_plugin_conference_themes()[theme].js_path
+        if not path:
```

Comment:
```suggestion
        if not (path := get_plugin_conference_themes()[theme].js_path):
```

Reply:
If we do it this way, we need to to query get_plugin_conference_themes()[theme].js_path a second time after the return None because we haven't actually set the path variable.

Reply:
look closely ;) the walrus operator assigns to `path`. just a more concise way of writing it

Reply:
Oh wow. This is the first I've heard of the walrus operator. Clever. It's what I get for going years in-between using python lol. Tested and can confirm it works.